ui-kit/src/main/kotlin/org/p2p/uikit/atoms/UiKitBadgeView.kt

```diff
@@ -0,0 +1,91 @@
+package org.p2p.uikit.atoms
+
+import android.content.Context
+import android.graphics.Paint
+import android.graphics.drawable.ColorDrawable
+import android.graphics.drawable.LayerDrawable
+import android.graphics.drawable.ShapeDrawable
+import android.graphics.drawable.shapes.RoundRectShape
+import android.util.AttributeSet
+import android.view.Gravity
+import androidx.appcompat.widget.AppCompatTextView
+import androidx.core.content.ContextCompat
+import org.p2p.uikit.R
+import org.p2p.uikit.utils.toPx
+
+private const val RADIUS_LIST_SIZE = 8
+private const val VERTICAL_PADDING_DP = 0
+private const val HORIZONTAL_PADDING_DP = 8
+
+class UiKitBadgeView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : AppCompatTextView(context, attrs, defStyleAttr) {
+
+    enum class Shape {
+        OVAL,
+        RECTANGLE
+    }
+
+    init {
+        context.obtainStyledAttributes(attrs, R.styleable.UiKitBadgeView).use { typedArray ->
+            val defaultTextColor = ContextCompat.getColor(context, R.color.mountain)
+            val hasStroke = typedArray.getBoolean(R.styleable.UiKitBadgeView_hasStroke, false)
+            val shape = if (typedArray.getInt(R.styleable.UiKitBadgeView_shape, 0) == 0) {
```

Comment:
Why you didn't make it as text? 
It would be easy to parsing, for example:
`Shape.valueOf(shape)` and you won't need `if` after it

Moreover, 1 or 0 is confusing, the developer doesn't no if 0 is `Oval` and 1 is `Rectangle`

Reply:
This attribute has limited values so for easy using it in xml it should be enum. But making custom `valueOf()` function is a nice point